source/php/Component/Nav/style/default.blade.php

```diff
@@ -8,6 +8,7 @@
     @icon([
       'icon' => $item['icon']['icon'] ?? null,
       'size' => $item['icon']['size'] ?? null,
+      'filled' => $item['icon']['filled'] ?? false,
```

Comment:
Should this value be false? In order to be able to style this trough the customizer the default should be null?

Reply:
Oh, thats right. Well caught!